src/components/filterPanel/FilterPanel.js

```diff
@@ -1,16 +1,30 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { useMatomo } from '@datapunt/matomo-tracker-react';
 import SVGIcon from 'components/SVGIcon/SVGIcon';
-import { SpotStatusTypes, SpotTypes } from 'constants.js';
+import { SpotStatusTypes, SpotTypes, Stadsdeel, endpoints } from 'constants.js';
 import { resetFilter } from 'components/map/helpers';
 import styles from './FilterPanel.module.scss';
-import { ContextMenuOptions } from './FilterPanel.constants';
+import { ContextMenuOptions, MenuOptions } from './FilterPanel.constants';
 import classNames from 'classnames';
 import { StatusDisplayNames, SpotTypeDisplayNames } from '../../constants';
 import SelectMenu from '../../shared/selectMenu/SelectMenu';
 import { ReactComponent as FilterIcon } from 'assets/icons/icon-filter.svg';
 import { ReactComponent as ChevronIcon } from 'assets/icons/chevron-top.svg';
+import { Button, themeSpacing } from '@datapunt/asc-ui';
+import styled from '@datapunt/asc-core';
+import useDownload from 'shared/hooks/useDownload';
+
+const ExportButton = styled(Button)`
+  margin: ${themeSpacing(2, 0)};
+`;
+
+const FilterWrapperStyle = styled.div`
+  max-height: calc(780px - 33px);
+  overflow-x: hidden;
+  overflow-y: auto;
+  will-change: transform;
```

Comment:
I don't see a `transform` style rule. Is this line necessary?

Reply:
removed